domains/Contact/ManageTasks/Services/UpdateContactTask.php

```diff
@@ -26,6 +26,7 @@ public function rules(): array
             'contact_task_id' => 'required|integer|exists:contact_tasks,id',
             'label' => 'required|string|max:255',
             'description' => 'nullable|string|max:65535',
+            'due_at' => 'nullable|date_format:Y-m-d',
```

Comment:
It would be easier to declare it as a `date`, so when calling the service, you can just pass the data, and then format it inside this service.

Reply:
But if we do that, it means we can pass it the date object no matter the format. So we assume we'll receive a Carbon object entirely? Dates can have multiple formats otherwise.

Reply:
If you use that:
```suggestion
            'due_at' => 'nullable|date',
```
the data can be a `DateTimeInterface` or a string that represents a date, which is fine, no?